modules/Bio/EnsEMBL/G2P/Utils/Downloads.pm

```diff
@@ -213,7 +218,9 @@ sub write_data {
     # map attrib ids to values
     my $confidence_category = ($confidence_category_attrib) ? $confidence_category_attribs->{$confidence_category_attrib} : 'No confidence category';
     my $allelic_requirement = ($ar_attrib) ? join(',', map { $allelic_requirement_attribs->{$_} } split(',', $ar_attrib)) : undef;
+    my $cross_cutting_modifier = ($ccm_attrib) ? $cross_cutting_modifier_attribs->{$ccm_attrib} : "No cross cutting modifier";
```

Comment:
We will now need to accomodate multiple cross-cutting modifiers, so will need to change this .